demo-circuit/src/blaze_csw/data_structures.rs

```diff
@@ -233,23 +288,44 @@ pub struct CswFtProverData {
 }
 
 impl CswFtProverData {
-    pub(crate) fn get_phantom(commitment_hashes_number: u32) -> Self {
+
+    pub(crate) fn get_default(commitment_hashes_number: u32) -> Self {
         Self {
             ft_output: CswFtOutputData::default(),
             ft_input_secret_key: [false; SIMULATED_SCALAR_FIELD_MODULUS_BITS],
-            mcb_sc_txs_com_start: PHANTOM_FIELD_ELEMENT,
+            mcb_sc_txs_com_start: FieldElement::default(),
+            merkle_path_to_sc_hash: GingerMHTBinaryPath::new(vec![
+                (FieldElement::default(), false);
+                CMT_MT_HEIGHT
+            ]),
+            ft_tree_path: GingerMHTBinaryPath::new(vec![
+                (FieldElement::default(), false);
+                FWT_MT_HEIGHT
+            ]),
+            sc_creation_commitment: FieldElement::default(),
+            scb_btr_tree_root: FieldElement::default(),
+            wcert_tree_root: FieldElement::default(),
+            sc_txs_com_hashes: vec![FieldElement::default(); commitment_hashes_number as usize],
+        }
+    }
+
+    pub(crate) fn get_phantom(commitment_hashes_number: u32) -> Self {
+        Self {
+            ft_output: CswFtOutputData::get_phantom(),
+            ft_input_secret_key: [false; SIMULATED_SCALAR_FIELD_MODULUS_BITS],
```

Comment:
Let's set it to an array of all 1s, like you did for the CswUtxoInputData

Reply:
Done with commit [cd72c64b705c3fc92f5fe40a8e540bbdeb9d523f](https://github.com/HorizenOfficial/zendoo-sc-cryptolib/commit/cd72c64b705c3fc92f5fe40a8e540bbdeb9d523f).